### DISCUSSION TITLE
Creating graphs -- should all be sageplot?

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/calculus#83] [Q&A]
Wondering if there is a 'right' way to generate graphs--maybe `<sageplot>`? I think @StevenClontz suggested this the other day.

Asking in particular as I see two issues #77 #78 that specifically are about fixing graphs, and not sure if we should fix what is there or begin homogenizing.  Right now I see some static images, and some tikz used in different places.

**Comment by StevenClontz** [comment]
I'm unavailable, but there's a discussion in tomorrow's [PROSE office hours](https://groups.google.com/g/pretext-announce/c/utnV2iR81HU) on work to create tactile graphics for Active Prelude to Calculus.

I'm going to open a pretext-support thread to get community thoughts on accessible graphics...

**Reply by StevenClontz** [reply]
https://groups.google.com/g/pretext-support/c/jaez1QFmbEE

**Reply by siwelwerd** [reply]
So it sounds like definitely do not put effort into remaking graphs at this point (the fix to the issues above seems to be the `xml:id`s, so no sense messing with the code to generate them).  From that discussion it is not really clear what advantage asymptote has over sageplot.

**Reply by StevenClontz** [reply]
I agree that investing in Asymptote is not a clearly better solution. Furthermore, the implementation of David Austin's graphics markup is apparently in Python, so SageMath/Python seem more promising to me.
